templates/docs/examples/patterns/_tooltips_script.js

```diff
@@ -0,0 +1,29 @@
+function handleTooltipEvents(tooltip) {
+  var trigger = document.querySelector('[aria-describedby="' + tooltip.getAttribute('id') + '"]');
+
+  trigger.addEventListener('mousemove', function (e) {
+    // move with the cursor, but offset it
+    // so it isn't obscured by the cursor
+    var offset = 20;
+
+    tooltip.style.top = e.y + offset + 'px';
+    tooltip.style.left = e.x - offset + 'px';
+  });
+
+  trigger.addEventListener('mouseout', function () {
+    tooltip.style.removeProperty('top');
```

Comment:
I agree `u-hide` is a good idea - I removed the properties explicitly so that even when hidden it's not technically positioned on screen (it will still have the `top` and `left` values set by the `mousemove` event listener), but maybe that's not so bad if it's not visible.